core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -97,13 +116,23 @@ public InProcessTelemetryChannel(String endpointAddress, boolean developerMode)
     public InProcessTelemetryChannel(Map<String, String> nameAndValues) {
         boolean developerMode = false;
         String endpointAddress = null;
+        int maxQueueItemCount = DEFAULT_NUMBER_OF_TELEMETRIES_PER_CONTAINER;
+        int sendIntervalInMillis = TRANSMIT_BUFFER_DEFAULT_TIMEOUT_IN_SECONDS;
 
         if (nameAndValues != null) {
             developerMode = Boolean.valueOf(nameAndValues.get(DEVELOPER_MODE));
-            endpointAddress = nameAndValues.get(EndpointAddress);
+            endpointAddress = nameAndValues.get(ENDPOINT_ADDRESS);
+            maxQueueItemCount = translateNumber(nameAndValues.get(MAX_TELEMETRY_ITEMS_IN_QUEUE),
```

Comment:
Please align the parameters either by moving the first parameter one line below or moving the rest of the parameters to be aligned with the first one (the same goes to the next call as well)